### PR TITLE
Update room.md

### PR DESCRIPTION
**Pull request by vrod9429**
removing 'managed' under booking type and adding floor label

### CODE REVIEW

File: api-reference/beta/resources/room.md [hunk 1]
```diff
@@ -28,11 +28,12 @@ In Exchange Online, each room is associated with a room mailbox. Derived from [p
...
 | audioDeviceName        | String                                            | Specifies the name of the audio device in the room. |
-| bookingType            | [bookingType](#bookingtype-values)                | Type of room. Possible values are `standard`, `managed`, and `reserved`. |
+| bookingType            | [bookingType](#bookingtype-values)                | Type of room. Possible values are `standard`, and `reserved`. |
 | building               | String                                            | Specifies the building name or building number that the room is in. |
 | capacity               | String                                            | Specifies the capacity of the room. |
 | displayName            | String                                            | The name associated with the room. |
 | displayDeviceName      | String                                            | Specifies the name of the display device in the room. |
+| floorLabel             | String                                            | Specifies a descriptive label for the floor, for example, P. |
```
Comment (angelgolfer-ms) [inline api-reference/beta/resources/room.md]: Please make sure you add the property to the JSON representation section as well.
You can find the CDK topic here: https://msgo.azurewebsites.net/add/document/manage-content/steps-add-properties-to-resource-topic.htm

Comment (angelgolfer-ms) [reply api-reference/beta/resources/room.md]: Please include the change to bookingType and the addition of floorLabel in your changelog item.

**Comment by Lauragra** [comment]
@vrod9429 , the property is live in the metdata; can you please get this PR ready to publish? Please resolve the merge conflict and add an entry to the changelog for the new property.

**Comment by angelgolfer-ms** [comment]
@Lauragra , if you check the live docs, the changes are live. @vrod9429 probably forgot and can close this PR.

**Pull request closed without merging**